sig-auth/governance.md

```diff
@@ -0,0 +1,153 @@
+# SIG Auth Governance
+
+## Roles
+
```

Comment:
What is the mission / scope / subprojects?

Reply:
+1 See[ the SIG charter template](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-charter-template.md) if you need some guidance

Reply:
I'm happy to help brainstorm this with y'all.